src/shared/components/mutationMapper/DriverAnnotationProteinImpactTypeBadgeSelector.tsx

```diff
@@ -229,7 +234,29 @@ export default class DriverAnnotationProteinImpactTypeBadgeSelector extends Prot
 
     protected get options() {
         // get options, hide "Other" if it's 0
-        return SELECTOR_VALUE_WITH_VUS.map(value => ({
+        let excludedProteinTypes = this.props.excludedProteinTypes || [];
+        if (this.props.counts) {
+            if (
+                this.props.counts[ProteinImpactType.OTHER_PUTATIVE_DRIVER] +
```

Comment:
We should probably just combine these 2 conditions

```suggestion
        if (this.props.counts && this.props.counts[ProteinImpactType.OTHER_PUTATIVE_DRIVER] +
```